web/modules/custom/dept_node/dept_node.module

```diff
@@ -294,3 +295,22 @@ function dept_node_entity_type_alter(&$entity_types) {
     }
   }
 }
+
+/**
+ * Implements hook_pathauto_alias_alter().
+ */
+function dept_node_pathauto_alias_alter(&$alias, array &$context) {
+  $dbConn = \Drupal::database();
+
+  $results = $dbConn->query("SELECT pa.path, pa.alias FROM path_alias pa WHERE pa.alias = :alias", [':alias' => $alias])->fetchCol();
```

Comment:
Curly braces on the table name recommended, so should be `{path_alias} pa`

Reply:
good spot !